packages/mobile/src/screens/sign-on-screen/screens/SelectArtistScreen/SelectArtistsScreen.tsx

```diff
@@ -22,21 +17,12 @@ import { SelectArtistsPreviewContextProvider } from './selectArtistPreviewContex
 
 const Tab = createMaterialCollapsibleTopTabNavigator()
 
-type SelectArtistsValues = {
-  selectedArtists: string[]
-}
-
-const initialValues: SelectArtistsValues = {
-  selectedArtists: []
-}
-
 export const SelectArtistsScreen = () => {
```

Comment:
Might be a good idea to add a comment somewhere in here explaining why we're not using Formik